payloads/src/payload/mod.rs

```diff
@@ -1,4 +1,312 @@
-mod payload;
+mod payload {
```

Comment:
I would like to recommend removing this `mod payload` nesting and removing the `pub use payload::*` at the bottom of the file.